examples/sv/nsv5613472_ins.json

```diff
@@ -0,0 +1,48 @@
+{   
+   "name": "nsv5613472",
+   "primary_source": "DGVa",
+   "alleles": [{
+        "name": "",
+        "alternative_names": [],
+            "type": "StructuralVariationAllele",
+            "allele_type" :
+            {
+                "accession_id": "SO:0001019"
+            },
+            "slice": {
+            "location": {
```

Comment:
For insertions how do we convey the number of alleles inserted?

Reply:
Added length to StructuralVariant and StructuralVariantAllele. Example needed for insertion